Lib/gftools/meta.py

```diff
@@ -0,0 +1,75 @@
+"""Generate meta table for a font
+
+This module exports the function "gen_meta_table" which can be used to
+add a meta table designating supported and/or design scripts/languages.
+"""
+from fontTools.ttLib import TTFont, newTable
+import os
+import logging
+import yaml
+import re
+from langcodes import Language
+from langcodes.tag_parser import LanguageTagError
+from langcodes.data_dicts import VALIDITY
+
+__all__ = ["gen_meta_table"]
+
+
+log = logging.getLogger(__name__)
+
+
+def gen_meta_table(ttFont, config):
```

Comment:
It may be nice to move this up a level to the fontTools builder

Reply:
I have no problems with keeping it here if there's little interest.

Reply:
This way allows for the possibility of adding a command-line utility to add the table, outside of the builder.